utils/unittest/fixtures.go

```diff
@@ -162,7 +184,7 @@ func WithoutSeals(payload *flow.Payload) {
 }
 
 func BlockWithParentFixture(parent *flow.Header) flow.Block {
-	payload := PayloadFixture(WithoutSeals)
+	payload := PayloadFixture()
```

Comment:
This change breaks a lot of unit tests in `protocol/badger/mutator_test.go`

Reply:
yeah, reverted